test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -164,7 +164,7 @@ class CheckYourAnswersControllerSpec extends SpecBase with MockitoSugar with Sca
     }
 
     "onSubmit" must {
-      "removeFRE and redirect to ConfirmationClaimStoppedController when submission success" in {
+      "removeFRE and redirect to YourEmployerController when submission success" in {
```

Comment:
This should go to confirmation stopped??? need to confirm this but didn't think we need to care about YourEmployer if only a removal